File: app/src/main/java/com/codenicely/services/feelbazaar/vendorapp/course_detail/provider/CourseDetailProvider.java
package com.codenicely.services.feelbazaar.vendorapp.course_detail.provider;

import com.codenicely.services.feelbazaar.vendorapp.helper.PresenterCallback;

public interface CourseDetailProvider {
    void requestCourseDetail(String url, PresenterCallback presenterCallback);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/common/provider/CourseDetailProvider.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.common.provider

import org.wikiedufoundation.wikiedudashboard.helper.PresenterCallback

interface CourseDetailProvider {
    fun requestCourseDetail(url: String, presenterCallback: PresenterCallback<*>)
}
